code/controllers/subsystem/spatial_gridmap.dm

```diff
@@ -290,10 +243,6 @@ SUBSYSTEM_DEF(spatial_grid)
 
 	. = list()
 
-	//cache for sanic speeds
-	var/cells_on_y_axis = src.cells_on_y_axis
-	var/cells_on_x_axis = src.cells_on_x_axis
```

Comment:
dumb, revert

Reply:
wait no this change isnt dumb, each of these datum vars are only accessed once per call so caching it to a proc var makes no sense